### ISSUE TITLE
Room name remains in title, even on home screen

### ISSUE THREAD

**Issue by ndarilek** [vector-im/element-web#18668]
### Steps to reproduce

1. Start from any Matrix room. Note the room name in the title.
2. Press Ctrl-Alt-H.
3. Note that you're now on Element's home screen, but the window/page title still indicates that you're in a room.

### What happened?

The room title remains in the page/window title.

### What did you expect?

The page/window title should be cleared to indicate that the home screen is active.

### Operating system

Windows

### Application version

1.8.1

### How did you install the app?

Official website

### Have you submitted a rageshake?

No
